components/Form/FormInput.js

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+const specialChars = /[`!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?~]/;
+
+// const regex = /[a-zA-Z0-9 .]+/;
+
+export const FormInput = ({
+  defaultValue,
+  onChange,
+  labelClassName,
+  label,
+  required,
+  value,
+  pattern,
+  className,
+  ...props
+}) => {
+  const [userInput, setUserInput] = useState(defaultValue);
+  const [isValid, setIsValid] = useState(true);
+
+  const inputHandler = (e) => {
+    if (onChange) {
+      onChange(e);
+    } else {
+      pattern && setIsValid(!specialChars.test(e.target.value));
+      setUserInput(e.target.value);
+    }
+  };
+
+  return (
+    <>
```

Comment:
Wouldn't it be better to wrap this in div? So it is always on top of each other no matter the setup outside?

Choose whatever you think is the best